Jenkinsfile

```diff
@@ -0,0 +1,61 @@
+@Library(value = "tomtom-nav-pipeline-shared-library@1.2.0") _
+
+pipeline {
+  agent {
+    label 'Linux'
+  }
+
+  options {
+    timestamps()
+    disableConcurrentBuilds()
+  }
+
+  environment {
+    GRADLE_OPTS = "-Dorg.gradle.daemon=false"
+    DEPLOYER_CREDENTIALS = credentials("bintray-tomtom-nav-pipeline")
+  }
+
+  stages {
+
+    stage("Build") {
+      steps {
+        skipBuildWithOnlyChangesIn(['CHANGELOG.md', 'Jenkinsfile'] as String[])
```

Comment:
Shouldn't you also skip in case of `README.md` as this is also not part of your package?